pkg/vere/io/http.c

```diff
@@ -985,6 +1056,9 @@ _http_rec_accept(h2o_handler_t* han_u, h2o_req_t* rec_u)
   }
   else {
     u3_hreq* req_u = _http_req_prepare(rec_u, _http_req_new);
+    if ( c3y == _http_req_cache(req_u) ) {
+      return 0;
```

Comment:
The early return here seems inconsistent to me.  I think this would be clearer like this:
```C
if ( c3n == _http_req_cache(req_u) ) {
  _http_req_dispatch(req_u, req);
}
```
That way the final `return 0` would run no matter which conditional branches were taken.